Firebase/Database/Api/FIRDatabaseComponent.m

```diff
@@ -35,6 +35,7 @@
     FIRDatabaseDictionary;
 
 @interface FIRDatabaseComponent () <FIRComponentLifecycleMaintainer, FIRComponentRegistrant>
+@property (nonatomic) FIRDatabaseDictionary *instances;
```

Comment:
This doesn't need to be `FIRDatabaseDictionary` anymore, since it's supposed to be keyed on app name. Since this can only be instantiated with a single app, it should just be `NSMutableDictionary<FRepoInfo *, FIRDatabase *> *`. `FIRDatabaseDictionary` can be removed entirely.